Makefile

```diff
@@ -10,7 +10,8 @@ INSTALL=install
 CFLAGS ?= -O2
 PICFLAG = -fPIC
 C99FLAG = -std=c99
-UCFLAGS = $(CFLAGS) $(PICFLAG) $(C99FLAG) -DUTF8PROC_EXPORTS
+WCFLAGS = -Wall -Wmissing-prototypes -pedantic
```

Comment:
Better suggestion: move `-Wmissing-prototypes` to `CFLAGS` in `.travis.yml`.

The flag serves a purpose, to warn when internal functions are accidentally exported.